data/wanderer/wanderers middle.txt

```diff
@@ -2847,9 +2879,54 @@ mission "Wanderers: Sestor Alt: Return to Wanderers"
 	on offer
 		log "Succeeded in destroying the Kor Sestor fleet and eliminating the Alpha base on the anarchist world of Zenith. He is very grateful that the bomb or super-weapon, whatever it was, did not need to be used."
 		conversation
+			branch "no Danforth reset"
+				"Danforth in mission sequence" != 3
+			action
+				"Danforth in mission sequence" = 0
+			label "no Danforth reset"
+			branch "waiting for help with Hai"
+				has "Hai Reveal [A12] Into the Lion's Den: declined"
 			`The Navy officers report the results of the battle on Zenith's surface to Danforth. "So it's over," he says. "Captain <last>, thank you for your assistance. This battle would have gone very differently if you were not here."`
 			`	Now that the Kor Sestor fleet has been dealt with, you should probably return and see how things are going for the Wanderers in Korath space.`
 				accept
+			label "waiting for help with Hai"
+			action
+				event "Danforth is available after Sestor invasion" 20
+			`The Navy officers report the results of the battle on Zenith's surface to Danforth. "So it's over," he says. "Captain <last>, thank you for your assistance. This battle would have gone very differently if you were not here.`
+			`	"Now that we've fought off one invasion, it's time to prevent a second. I'll let you know when the Navy is ready for you to start a dialog with the Hai. In the meantime, you might want to check on the Wanderers."`
```

Comment:
Dialog? That doesn’t quite feel like the right word.

Reply:
He is using formal terminology.